src/commands/publish/mod.rs

```diff
@@ -105,47 +116,32 @@ fn upload_script(
     Ok(())
 }
 
-fn deploy(user: &GlobalUser, target: &Target) -> Result<(), failure::Error> {
-    let routes = target.routes()?;
-    log::info!("routes: {:#?}", &routes);
+fn deploy(user: &GlobalUser, deploy_target: &DeployTarget) -> Result<(), failure::Error> {
+    match deploy_target {
+        DeployTarget::Zoneless(zoneless_config) => {
+            // this is a zoneless deploy
+            log::info!("publishing to subdomain");
```

Comment:
might be worth adding the subdomain being published to

Reply:
we do print after, but we don't have the subdomain locally, we have to query it in the course of the actual `publish_to_subdomain` function.